pkg/costmodel/aggregation.go

```diff
@@ -2247,7 +2247,12 @@ func (a *Accesses) ComputeAllocationHandler(w http.ResponseWriter, r *http.Reque
 	// AccumulateBy is an optional parameter that accumulates an AllocationSetRange
 	// by the resolution of the given time duration.
 	// Defaults to 0. If a value is not passed then the parameter is not used.
-	accumulateBy := qp.GetDuration("accumulateBy", 0)
+	accumulateBy := kubecost.AccumulateOption(qp.Get("accumulateBy", ""))
```

Comment:
What do you think of the idea of lumping this all under `accumulate`? Don't worry about backwards-compatibility. Just in terms of an API, something like:
```
accumulate={false,true,hourly,daily,weekly,monthly}
```
instead of two params?

Reply:
Just thinking that we already have `aggregate` as a param, and having a single `accumulate` might mirror that nicely. Not a big deal either way, though, to be clear.

Reply:
In that case, we'd want to make sure that we had a little ParseAccumulateBy func that helped to map "true" to "all", "false" to "", etc.

Reply:
That is a cleaner solution.